imports/plugins/core/ui/client/containers/inventoryBadge.js

```diff
@@ -0,0 +1,32 @@
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { InventoryBadge } from "../components/badge";
+
+const composer = (props, onData) => {
+  const { variant, soldOut } = props;
+  const { inventoryManagement, inventoryPolicy, lowInventoryWarningThreshold, inventoryQuantity } = variant;
+  let label = null;
+  let i18nKeyLabel = null;
+  let status = null;
+  // TODO: update this to use Catalog API.
+  if (inventoryManagement && !inventoryPolicy && Number(inventoryQuantity) === 0) {
```

Comment:
Why do we have to keep converting these to `Number`? Shouldn't they just be a number? What happens if we can an actual string there?

Reply:
Yes, you are correct these should be numbers. The problem is with the components used for getting the quantities. Working to replace them.